mllib-tests/v1p4/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -75,11 +78,12 @@ abstract class RegressionAndClassificationTests[M](sc: SparkContext) extends Per
 abstract class GLMTests(sc: SparkContext)
   extends RegressionAndClassificationTests[GeneralizedLinearModel](sc) {
 
-  val STEP_SIZE =      ("step-size",   "step size for SGD")
-  val NUM_ITERATIONS = ("num-iterations",   "number of iterations for the algorithm")
-  val REG_TYPE =       ("reg-type",   "type of regularization: none, l1, l2")
-  val REG_PARAM =      ("reg-param",   "the regularization parameter against overfitting")
-  val OPTIMIZER =      ("optimizer", "optimization algorithm: sgd, lbfgs")
+  val STEP_SIZE =         ("step-size",   "step size for SGD")
+  val NUM_ITERATIONS =    ("num-iterations",   "number of iterations for the algorithm")
+  val REG_TYPE =          ("reg-type",   "type of regularization: none, l1, l2, elastic-net")
+  val ELASTIC_NET_PARAM = ("elastic-net-Param",   "elastic-net param, 0.0 for L2, and 1.0 for L1")
+  val REG_PARAM =         ("reg-param",   "the regularization parameter against overfitting")
+  val OPTIMIZER =         ("optimizer", "optimization algorithm: sgd, lbfgs")
```

Comment:
elasticnet is using `lbfgs/owlqn`, but lbfgs is also used in old mllib LoR. I will probably document it.